Update terminology from 'master key' to 'database master key' (#10356)
MicrosoftDocs/sql-docs: docs/relational-databases/security/encryption/back-up-a-database-master-key.md

    ```  
  
    > [!NOTE]  
    > The file path to the key and the key's password (if it exists) will be different than what is indicated above. Please make sure that both are specific to your server and key set-up.
    > [!NOTE]  

>     > TThe password is required for a backup. Please be sure this password is stored safely and securely.

Reviewer: ```suggestion
    > The password is required for a backup. Make sure this password is stored safely and securely.
```
